docs/ProfilerOptions.md

```diff
@@ -0,0 +1,113 @@
+| `-i --interval N`  | `interval=N`      | Interval has different meaning depending on the event. For CPU profiling, it's CPU time. In wall clock mode, it's wall clock time. For Java method profiling or native function profiling, it's number of calls. For PMU profiling, it's number of events.<br/>Example: `asprof -e cpu -i 500us 8983`                                                                                                            |
+| `--alloc N`        | `alloc=N`         | Allocation profiling interval in bytes or in other units, if N is followed by `k` (kilobytes), `m` (megabytes), or `g` (gigabytes).                                                                                                                                                                                                                                                                              |
+| `--live`           | `live`            | Retain allocation samples with live objects only (object that have not been collected by the end of profiling session). Useful for finding Java heap memory leaks.                                                                                                                                                                                                                                               |
+| `--lock DURATION`  | `lock=DURATION`   | In lock profiling mode, sample contended locks when total lock duration overflows the threshold                                                                                                                                                                                                                                                                                                                  |
+| `-j N`             | `jstackdepth=N`   | Sets the maximum stack depth. The default is 2048.</br>Example: `asprof -j 30 8983`                                                                                                                                                                                                                                                                                                                              |
+| `-I include`       | `include=PATTERN` | Filter stack traces by the given pattern(s). `-I` defines the name pattern that *must* be present in the stack traces. `-I` can be specified multiple times. A pattern may begin or end with a star `*` that denotes any (possibly empty) sequence of characters.</br>Example: `asprof -I 'Primes.*' -I 'java/*' 8983`                                                                                           |
```

Comment:
To make it consistent, parameter names should be called similarly in both variants, e.g. `-I PATTERN` and `include=PATTERN`.

Reply:
Addressed in [6526389](https://github.com/async-profiler/async-profiler/pull/1029/commits/65263898accd8b0d26d286d63e4f1ef990ad9fbf)